website/cue/reference/remap/functions/encode_zlib.cue

```diff
@@ -0,0 +1,37 @@
+package metadata
+
+remap: functions: encode_zlib: {
+	category:    "Codec"
+	description: """
+		Encodes the `value` to [Zlib](\(urls.zlib)).
```

Comment:
Should we be describing these as encode/decode or compress/uncompress?

Reply:
Well, actually I did it in the same way it's done in the previous PR: https://github.com/vectordotdev/vector/blob/master/website/cue/reference/remap/functions/encode_gzip.cue#L6

I think both variants are valid since the compression process is some kind of encoding. If we want to change it here to compress/decompress, we also need to change it in other places for being consistent.

At least, "encode/decode" words are consistent with the current function names.

Reply:
I'd be fine with encode/decode if only to avoid confusion in the docs/categories/etc - we can always make a new compression category in the future.